packages/components/src/__future__/Tag/Dismiss.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { Icon } from "@kaizen/component-library/components/Icon"
+import clearIcon from "@kaizen/component-library/icons/clear-white.icon.svg"
+
+export type DismissProps = {
+  onDismiss?: React.MouseEventHandler<HTMLButtonElement>
+}
+
+export const Dismiss = ({ onDismiss }: DismissProps): JSX.Element => (
```

Comment:
We're going to have to remove this too since its interactive we'll have to include more tests to support it, and we already have a well tested [ClearButton](https://cultureamp.design/?path=/docs/components-utilities-forms-clear-button--docs) component, just that it's currently too big, so we are weighing our options to resize that component and re-use it here to keep consistency.